app/models/user_credential.rb

```diff
@@ -1,7 +1,7 @@
 class UserCredential < ActiveRecord::Base
   MODES = %w[text java_script]
 
-  attr_accessible :credential_name, :credential_value, :mode
+  attr_accessible :credential_name, :credential_value, :mode, :id, :user_id, :created_at, :updated_at
```

Comment:
This is dangerous, making `user_id` accessible allows to override credentials of different users.